packages/fx-core/src/component/generator/copilotExtension/generator.ts

```diff
@@ -80,20 +77,10 @@ export class CopilotExtensionGenerator extends DefaultTemplateGenerator {
         inputs.targetFramework,
         inputs.placeProjectFileInSolutionDir === "true"
       ),
-      DeclarativeCopilot: isDeclarativeCopilot ? "true" : "",
+      DeclarativeCopilot: "true",
```

Comment:
Ideally this is not needed, but depends on https://github.com/OfficeDev/teams-toolkit/pull/13134